src/components/motebehov/MulighetForArbeid.tsx

```diff
@@ -61,42 +54,64 @@ const StyledSuccessStroke = styled(CheckmarkIcon)`
   font-size: 1.5rem;
 `;
 
-const StyledMulighetForArbeid = styled.div`
-  margin-top: 1em;
-`;
-
 interface AktivitetIkkeMuligProps {
   beskrivelse?: string;
   ikkeMuligListe: string[];
+  ikkeMuligGrunn: IkkeMuligGrunn;
   tittel: string;
 }
 
+export enum IkkeMuligGrunn {
+  MedisinskeArsaker,
+  ForholdPaArbeidsplassen,
+}
+
+const Icon = ({ ikkeMuligGrunn }: { ikkeMuligGrunn: IkkeMuligGrunn }) => {
+  switch (ikkeMuligGrunn) {
+    case IkkeMuligGrunn.MedisinskeArsaker:
+      return (
+        <FirstAidKitIcon
+          title="a11y-title"
+          fontSize="1.5rem"
+          className="mr-2"
+        />
+      );
+    case IkkeMuligGrunn.ForholdPaArbeidsplassen:
+      return (
+        <BriefcaseIcon title="a11y-title" fontSize="1.5rem" className="mr-2" />
```

Comment:
Endre `a11y-title` til noe som er a11y-vennlig? Evt kunne ikonet vært skjult for skjermlesere også 🤔

Reply:
Bra catch! Blind copy/paste her 🙈